.github/workflows/security_veracode_pipeline_scan.yml

```diff
@@ -0,0 +1,15 @@
+name: Security veracode pipeline scan
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "35 9 * * MON-FRI" # Every weekday at 09:35 UTC
+
+jobs:
+  security-veracode-pipeline-scan:
+    name: Project security veracode pipeline scan
+    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_veracode_pipeline_scan.yml@v2 # WORKFLOW_VERSION 
+    with:
+      channel_id: ${{ vars.SECURITY_ALERTS_SLACK_CHANNEL_ID || 'NO_SLACK' }}
+      docker_image_app_dir: '/opt/app'
```

Comment:
is this the right directory?

https://github.com/ministryofjustice/hmpps-trivy-discovery/blob/main/Dockerfile#L13C9-L13C13